package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "ngx-store",
     "private": true,
-    "version": "1.4.1",
+    "version": "6.0.0",
```

Comment:
Btw do you know will library compiled with A6 work for A5?

Reply:
It will probably not since this relies on RxJS 6. I would keep the 1.x series for A5 if desirable, and use the new master for A6.

All angular packages synchronized to to 6.0.0. It seems that is a new pattern they try to establish. I think it makes a lot of sense and as [the comment](https://github.com/zoomsphere/ngx-store/issues/43#issuecomment-383281610) in #43 shows there are others who would prefer this associated versioning.

But this is up to you, I just see merrit and propose this with this PR. It also makes it easy and clear to support multiple Angular LTS branches with associated version numbers.